docs/tutorial/3_the_dataset.md

```diff
@@ -152,33 +150,24 @@ A useful feature of the `Dataset` class, which it inherits from `NDCube` is the
 
 For example, to extract the Stokes I component of the dataset we would do:
 
-```{code-cell} python
+```{code-cell} ipython3
 ds[0]
 ```
 
 this is because the stokes axis is the first array axis, and the "I" profile is the first one (0 indexing).
 
 Note how we have dropped a world coordinate, this information is preserved in the `.global_coords` attribute, which contains the coordinate information which applies to the whole dataset:
 
-```{code-cell} python
+```{code-cell} ipython3
 ds[0].global_coords
 ```
 
 We can also slice the data further, as we would for a normal numpy array.
 So for instance we can select a small section of the image in Stokes I at some arbitrary wavelength:
 
-```{code-cell} python
-cropped = ds[0, 400:850, 100, 950:1600]
+```{code-cell} ipython3
+cropped = ds[0, 200:300, 100, 950:1600]
 cropped
 ```
 
 Notice again that this has reduced the dimensionality of the world coordinates as well as of the data itself.
-
-Finally, we can plot our cropped dataset to show the feature we were looking at.
-
-```{code-cell} python
-import matplotlib.pyplot as plt
-
-cropped.plot()
-plt.show()
```

Comment:
We're not showing the data here any more?

Reply:
Well we haven't downloaded it yet, and it's slow so I thought we could drop it?